build/rbac/rbac.yaml

```diff
@@ -757,6 +757,9 @@ spec:
     # Ceph mgr Prometheus Metrics
     - min: 9283
       max: 9283
+    # port for CSIAddons
+    - min: 9070
+      max: 9070
```

Comment:
This would limit, port for node-plugin csiaddons sidecar to just one port ?

Reply:
yes. Fixed it for now. Maybe can change it later.

Reply:
Looking at the IANA page https://www.iana.org/assignments/service-names-port-numbers/service-names-port-numbers.xhtml?&page=114, these ports are not allocated officially `9061-9079		Unassigned`.

So it's probably safe to use `9070`. We need to be careful since hostnetworking is enabled and hopefully we won't conflict with another service.